js/libs/form.js

```diff
@@ -153,8 +153,24 @@ Fliplet().then(function() {
       });
     }
 
+    function saveProgress() {
```

Comment:
Can you define this so that we're not repeating the logic in https://github.com/Fliplet/fliplet-widget-form-builder/blob/master/js/libs/form.js#L1015-L1025 ?